### ISSUE TITLE
1.9.0: New title page problems

### ISSUE THREAD

**Issue by Nyhde** [piersdeseilligny/betterfountain#152]
Hi there,

Firstly, thank you for a wonderful extension! I've very much enjoyed using it for the past few months, and have already written four short films using it. It has pretty much everything I want from a Fountain extension, and even some features I hadn't even thought of before stumbling upon them in this one.

However, the latest update seems to have introduced some problems on the new title page.

Here are the things that bother me:

1) The centering of all the supposedly centered items seems to be off by one or two characters.
2) At least Title:, Credit: and Author: seem to print too low in the PDF compared to ones rendered in earlier versions.
3) Notes: and Copyright: used to stay nicely on top of each other. As well as Draft Date: and Contact:. Now there's an extra space between each.

The new added keys like TL and TR and link support seem like nice additions, but if they mess with the basic layout, I would prefer it if they were reverted. The optimal solution would of course be to leave the new keys alone, but to revert the changed basic keys like Title: and Author: and such to their 1.8.11 status. That version works without any issues.

I hope this can be remedied, but if not, at least we have the versio 1.8.11.

Best regards
Nyhde

**Comment by piersdeseilligny** [comment]
Hi, thanks for the feedback and I'm glad you're enjoying the extension! I'il answer your concerns individually:

1. This is because the left margin on the title page (in 1.9.0) was the same as in the rest of the document, at 1.5" rather than 1" in order to leave space for the three-hole punch. However I looked into it and there doesn't really seem to be a consistent industry standard for left margins on the title page: some have it, some don't. But because I agree with you it looks nicer when it's absolutely centered, I've reverted this change in 1.9.1, currently in the marketplace

2. In versions prior to 1.9.0, the central text was at an arbitrary distance from the top of the document, regardless of how much text was above or below, or how long the central text was. After 1.9.0, the central text is always vertically centred between text above (only specified by `tl:` `tc:` or `tr:`) and text below (such as notes). This avoids text overlapping. However, if you add empty `cc:` elements, it'll move the central text higher up, as such:
```
Title: **A MOVIE TITLE**
Credit: Written by
Author: Some person
cc:
cc:
```
those empty lines will show up as "undefined" in the live preview, but will just be empty when rendered to PDF.

3. Unfortunately I was a little torn about what to do, because the spacing was inconsistent before. In the end I just decided to add an empty line between all separate elements, although I would have liked if it were only half a line, but that's actually oddly difficult to do with the way it currently works. As an alternative, you could include the copyright notice in the same element as the notes:
```
Title: **A MOVIE TITLE**
Credit: Written by
Author: Some person
Notes: some notes about the screenplay
       Copyright 2021
```
Or maybe more elegantly, in a BL element:
```
Title: **A MOVIE TITLE**
Credit: Written by
Author: Some person
bl: Some notes about the screenplay
    Copyright 2021
```


So to summarize, I've fixed problem number 1, so text is now perfectly centered, but 2 & 3 currently require workarounds. Sorry about these changes, but hopefully you understand why they were made and they aren't too big of an issue for you!

**Comment by Nyhde** [comment]
Thank you for your quick reply and update!

With the workaround of adding extra CC keys below the title, there's just enough flexibility for me to get the layout to look the way I want. The below texts don't really seem to matter that much after all, but the off-center title was bothering me quite a bit, and I'm very glad you fixed it. It's ultimately a small detail, but it can be these sorts of details that can make a screenplay fail before it's even been read.

Thank you again! I wonder, do you have a PayPal?

**Comment by piersdeseilligny** [comment]
@Nyhde Glad I could help! And yeah I totally agree about the importance of details - if ever you notice anything else let me know! And thank you, I do indeed have a Paypal :) https://www.paypal.com/paypalme/piersdeseilligny/
